config/core/configmaps/config-observability.yaml

```diff
@@ -0,0 +1,149 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-observability
+  namespace: google-sources
+data:
+  _example: |
```

Comment:
this example has outdated stuff.
Can you copy it from https://github.com/google/knative-gcp/blob/1c3f08ea3c07ede4a2de3c1c6db2931a60a4bf4c/config/core/configmaps/observability.yaml#L27